packages/js/src/ui/config/appearanceKeys.ts

```diff
@@ -133,4 +133,7 @@ export const appearanceKeys = [
 
   // Preferences Loading
   'preferencesLoadingContainer',
+
+  // Markdown
+  'markdownBold',
```

Comment:
This will allow more styling on text marked as bold.

Reply:
Maybe

```suggestion
  'strong',
```

Reply:
I want to specify that this is for markdown rendered elements. `strong` could mean everything that is "bold" in our components.